src/lib/models/subscriptions.rs

```diff
@@ -20,17 +41,19 @@ pub struct Subscription {
     pub aic_id: Option<Uuid>,
     pub aic_expires: Option<OffsetDateTime>,
     pub cj_event_value: Option<String>,
-    pub status: Option<String>,
-    pub status_history: Option<JsonValue>,
+    // Note we use string and json to save in database for simplicity
```

Comment:
Can you clarify what this means from a database perspective? Does "string" correspond to a text field rather than char?

Reply:
I mean as opposed to using an enum field that directly maps to our potential statuses.

Reply:
In theory this could be Option<Status>. But my brief efforts to try were not pretty. And the inflexibility of db level enums seems like it's not an obvious win. Maybe this comment is actually confusing.

Reply:
I agree. I have gone down the path of db-level enums in the past and I have found they are almost always (maybe always always) way more trouble than they're worth. Maybe would be good to just change the comment slightly to clarify.

```suggestion
    // Note we use strings and json, not enums, in the database for simplicity
```